website/docs/core-concepts/stacks/imports.md

```diff
@@ -5,41 +5,200 @@ sidebar_label: Imports
 id: imports
 ---
 
-Imports are how we reduce duplication of configurations by creating reusable baselines. The imports should be thought of almost like blueprints. Once a reusable catalog of Stacks is exists, robust architectures can be easily created simply by importing those blueprints.
+Imports are how we reduce duplication of configurations by creating reusable baselines. The imports should be thought of almost like blueprints. Once
```

Comment:
Not sure what's causing these line breaks but it makes it harder to review these lines of code. Is this happening with your editor? Or some automation?